doc/changes/unreleased.md

```diff
@@ -4,4 +4,7 @@
 
 ## Security Issues
 
-### Bug fixing
```

Comment:
Please mention related github issues in the changelog 
Please annotate GitHub PR with "Closes"
Candidates: 
* #153 
* #101 
* #159